static/js/floorplan.js

```diff
@@ -158,13 +164,31 @@ const FloorplanScreen = {
               break;
           }
         }
+        thing.element.dataset.index = this.things.length;
+        this.addInteract(thing);
         this.things.push(thing);
         // Dynamically layout all things including the one we just added
         this.updateVmin();
       });
     });
   },
 
+  addInteract: function(thing) {
+    thing.element.addEventListener('click', this.blackHole);
+    thing.element.addEventListener('mousedown', this.onPointerDown);
+    thing.element.addEventListener('mouseup', this.onPointerUp);
+    thing.element.addEventListener('touchstart', this.onPointerDown);
+    thing.element.addEventListener('touchend', this.onPointerUp);
```

Comment:
One of these events seems to be firing on right click, so that right clicking an icon turns it on and off (or navigates to the detail view in the case of custom things). This probably shouldn't happen.